packages/ui/components/VisualEditor.tsx

```diff
@@ -43,6 +37,14 @@ export const VisualEditor: React.FC<VisualEditorProps> = ({ schema, onSchemaChan
     onSchemaChange(newSchemaString);
   };
 
+  const selectStyle = {
+    backgroundColor: '#0F172A',
+    color: getColorForType(schemaObject.type || 'white', schemaObject.items?.type || 'red'),
+    borderRadius: '3px',
+    fontSize: '12px',
+    fontFamily: 'Inter, sans-serif'
+  };
+
   const renderRootTypeSelector = () => (
```

Comment:
for some reason this doesn't work on my machine. and selecting `array` gives an error.